tensorflow_tts/bin/preprocess.py

```diff
@@ -289,13 +290,10 @@ def preprocess():
     # define map iterator
     def iterator_data(items_list):
         for item in items_list:
-            sample = processor.get_one_sample(item)
```

Comment:
in baker processor, if some text has some problem, i will return None, so this line should keep

Reply:
ok :D

Reply:
i will merge korea first :D

Reply:
i have push this